<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Accession</class>
 <widget class="QDialog" name="Accession">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="ID_Edit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="PW_Edit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="NAME_Edit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>160</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="ID_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>56</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>아이디</string>
   </property>
  </widget>
  <widget class="QLabel" name="PW_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>56</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>비밀번호</string>
   </property>
  </widget>
  <widget class="QLabel" name="NAME_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>56</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>닉네임</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Accession_Button">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>230</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>가입</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
